llm_api_hub/server/app.py

```
@@ -348,6 +358,41 @@ def health() -> Dict[str, Any]:
+        raise HubError(501, "provider checklist is not available", code="not_implemented")
+    return provider_checklist.public_view(provider_checklist.load_state())
+
+
+class TransitionRequest(BaseModel):
+    state: str
+    account: Optional[str] = None
+    reason: Optional[str] = None
+
+
+@app.post("/v1/providers/{provider_id}/transition")
+def transition_provider(provider_id: str, request: TransitionRequest, raw_request: Request) -> Dict[str, Any]:
```

Pylint (4.0.6)

[convention] Line too long (110/100)

(C0301)